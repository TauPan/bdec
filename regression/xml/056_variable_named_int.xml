<protocol>
    <sequence name="a">
        <reference name="b" type="int" />
        <reference name="c" type="int" />
        <field name="d" length="${c} * 8"  type="text" />
    </sequence>

    <common>
        <field name="int" length="8" type="integer" />
    </common>
</protocol>
