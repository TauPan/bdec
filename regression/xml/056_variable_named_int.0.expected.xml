<a>
    <b>4</b>
    <int>3</int>
    <d>abcd</d>
    <e>zyx</e>
</a>
